gnd/src/main/java/com/google/android/gnd/ui/map/MapPolygon.java

```diff
@@ -40,6 +40,29 @@ public static Builder newBuilder() {
   @Override
   public abstract Feature getFeature();
 
+  public abstract Builder toBuilder();
```

Comment:
Should this class be renamed to `MapPolyline` for correctness?

Reply:
SGTM. Added a TODO for future refactoring.